src/commands/openapi/upload.ts

```diff
@@ -100,6 +100,7 @@ export default class OpenAPIUploadCommand extends BaseCommand<typeof OpenAPIUplo
     let status: APIUploadStatus = 'pending';
 
     while (this.isStatusPending(status) && count < 10) {
+      // biome-ignore lint/nursery/noAwaitInLoop: We need to wait between requests to avoid hitting rate limits.
```

Comment:
I just think Biome is neat.

![mqdefault](https://github.com/user-attachments/assets/f26f05db-16e5-4804-8248-594d5f7e4071)